test/ruby/test_thread.rb

```diff
@@ -1553,4 +1553,37 @@ def test_pending_interrupt?
     assert_equal(true, t.pending_interrupt?(Exception))
     assert_equal(false, t.pending_interrupt?(ArgumentError))
   end
+
+  def test_deadlock_backtrace
+    bug21127 = '[ruby-core:120930] [Bug #21127]'
+
```

Comment:
```suggestion
```

Probably only need one empty line here.

Reply:
Sorry for the mistake 🙈 I'll commit your suggestion.